rs/crypto/internal/crypto_lib/threshold_sig/bls12_381/src/ni_dkg/fs_ni_dkg/nizk_sharing.rs

```diff
@@ -313,16 +313,29 @@ pub fn verify_sharing(
         // LHS = product [C_i ^ x^i | i <- [1..n]]^x' * Y
         // RHS = product [y_i ^ x^i | i <- 1..n]^z_r * g_1^z_alpha
```

Comment:
As we are rearranging the verification equations to do a single multiexp, we should update this doc so that it matches the new lhs and rhs below. E.g.

```suggestion
        // Original relation:
        //   (∏_{i=1}^n C_i^{x^i})^{x'} * Y  ==  (∏_{i=1}^n y_i^{x^i})^{z_r} * g_1^{z_α}
        //
        // Equivalently, we can rewrite it by moving terms to opposite sides:
        //
        //   lhs = (∏_{i=1}^n C_i^{x^i})^{x'} * (∏_{i=1}^n y_i^{x^i})^{-z_r}
        //   rhs = g_1^{z_α} * Y^{-1}
```

We could simplify the lhs further as ∏_{i=1}^n (C_i^{x'}y_i^{-z_r})^{x^i}), but the version above is closer to the actual computation performed, so maybe better overall.